src/components/Tree/Tree.tsx

```diff
@@ -343,7 +352,9 @@ export const Tree = memo(
         };
 
         const handleDragMove = ({ delta }: DragMoveEvent) => {
-            setOffset(delta.x);
+            if (delta.x % 10 === 0) {
```

Comment:
Why 10? Maybe extracting it into a constant would explain better the magic number, wouldn't it?

Reply:
yes, or making it relative to the tree padding. The point is reducing the times we refresh the offset which was triggering useless re-renderings

Reply:
updated!